jujugui/static/gui/src/app/config-debug.js

```diff
@@ -53,10 +53,10 @@ var juju_config = { // eslint-disable-line no-unused-vars
   // There is also a hotkey to toggle the simulator.
   simulateEvents: false,
   readOnly: false,
-  // Set the GA_key to enable Google Analytics usage and calls. Also implies
-  // using cookies. For the debug configuration, the GA_key should be blank to
-  // prevent muddying the gathered statistics.
-  GA_key: '',
+  // Set the GTM_enabled to enable Google Tag Manager usage and calls. Also
+  // implies using cookies. For the debug configuration, the GTM_enabled should
+  // be false to prevent muddying the gathered statistics.
+  GTM_enabled: false,
```

Comment:
I think we can actually remove config-debug.js and config-prod.js, can't we? @hatched?